t/ui/12-needle-edit.t

```diff
@@ -142,8 +142,10 @@ sub add_needle_tag(;$) {
 
 sub add_workaround_property() {
     $driver->find_element_by_id('property_workaround')->click();
+    $driver->find_element_by_id('input_workaround_desc')->send_keys("workaround reason");
     wait_for_ajax;
-    is($driver->find_element_by_id('property_workaround')->is_selected(), 1, "workaround property selected");
+    is($driver->find_element_by_id('property_workaround')->is_selected(),    1, "workaround property selected");
+    is($driver->find_element_by_id('input_workaround_desc')->is_displayed(), 1, "workaround description displayed.");
```

Comment:
The strings are quoted inconsistently with `"..."` and `'...'`, even though there is no interpolation required. Same for punctuation in test descriptions.

Reply:
I will fix it.